app/models/member.rb

```diff
@@ -125,6 +125,10 @@ def self.can_be_member_of?(entity_or_class)
     ALLOWED_ENTITIES.include?(checked_class)
   end
 
+  def self.newest_first
+    order "#{table_name}.created_at DESC"
```

Comment:
`order` takes a hash. There's no need to use raw SQL here.

```suggestion
    order(created_at: :desc)
```